.github/workflows/go-function-offline-testing.yml

```
@@ -14,6 +14,7 @@ jobs:
       matrix:
         tests: [
           { directory: 'cors-go', go-version: '1.19' },
+          { directory: 'go-hello-world', go-version: '1.20' }
```

nit: this function uses the 1.12 go runtime
```suggestion
          { directory: 'go-hello-world', go-version: '1.21' }
```